libs/sdk/src/__test-utils__/fixtures/flow.fixtures.ts

```
@@ -0,0 +1,438 @@
+          id: 'google',
+          name: 'Google',
+          auth: { mode: 'transparent', remote: { provider: 'https://accounts.google.com' } },
+        },
+        { id: 'github', name: 'GitHub', auth: { mode: 'transparent', remote: { provider: 'https://github.com' } } },
+      ],
+    }),
+
+  /**
+   * Single app incremental auth scenario
+   */
+  incrementalAuth: (appId: string = 'slack') =>
```

ESLint

[error] Type string trivially inferred from a string literal, remove type annotation.

(@typescript-eslint/no-inferrable-types)